### DISCUSSION TITLE
Examples in TanStack Router are not compiling

### DISCUSSION THREAD

**Discussion by 3bdallahz** [TanStack/router#557] [Q&A]
Hi everyone,
I was trying to TanStack Router, and I got type errors like Outlet is not found in package ..
so I tried to see how the examples are doing it and non of the examples worked, they all failed
https://tanstack.com/router/v1/docs/examples/react/basic

Does anyone see this issue and/or how to fix it?

**Comment by livevsonline** [comment]
Try to install react-router :)

This is the latest prerelease, just add it to your package.json :)

"@tanstack/react-router": "^0.0.1-beta.83",
